pyaedt/edb_core/components.py

```diff
@@ -783,22 +786,40 @@ def create_port_on_pins(self, refdes, pins, reference_pins, impedance=50.0, port
         if not len([pin for pin in reference_pins if isinstance(pin, EDBPadstackInstance)]) == len(reference_pins):
             return
         if len(pins) > 1:
+            pec_boundary = False
+            self._logger.info(
+                "Disabling PEC boundary creation, this feature is supported on single pin "
+                "ports only, {} pins found".format(len(pins))
+            )
             group_name = "group_{}_{}".format(pins[0].net_name, pins[0].name)
             pin_group = self.create_pingroup_from_pins(pins, group_name)
             term = self._create_pin_group_terminal(pingroup=pin_group, term_name=port_name)
 
         else:
-            term = self._create_terminal(pins[0], term_name=port_name)
+            term = self._create_terminal(pins[0].primitive_object, term_name=port_name)
         term.SetIsCircuitPort(True)
         if len(reference_pins) > 1:
+            pec_boundary = False
+            self._logger.info(
+                "Disabling PEC boundary creation, this feature is supported on single pin"
+                "ports only, {} reference pins found".format(len(reference_pins))
```

Comment:
```suggestion
                "Disabling PEC boundary creation. This feature is supported on single pin"
                "ports only {} reference pins found.".format(len(reference_pins))
```
Message strings should be complete sentences--so please revise the preceding content to this style guideline.